README.md

```diff
@@ -35,4 +28,4 @@ specification release are tagged using the following format:
 imply the same quality level as SDK tags. SDK tags follow the
 `sdk-<`_`version`_`>.<`_`patch`_`>` format (e.g., `sdk-1.1.92.0`).
 
-This scheme was adopted following the 1.1.96 Vulkan specification release.
+This scheme was adopted following the `1.1.96` Vulkan specification release.
```

Comment:
150 releases on, I think we could drop this sentence tool.

Reply:
:+1: